src/components/Task.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
```

Comment:
I suggest changing the component name to `Todo`.

Reply:
Resolved on ce60b5e0507612845215b501620be1ff1c8de067